src/controllers/permissionController.js

```diff
@@ -0,0 +1,118 @@
+import { Role, Permission } from '../sequelize/models';
+
+/**
+ * @description Users role and thier permission
+ */
+class permissionController {
+  /**
+   *
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} return object containing a new created permission
+   */
+  static async createPermision(req, res) {
+    const { role } = req.params;
+    const newPermission = req.body.permission;
+
+    const { id } = await Role.findOne({ where: { role } });
+    const permission = await Permission.findOne({ where: { roleId: id } });
+
+    if (permission && permission.roleId === id && permission.permission === newPermission) {
+      return res.status(409).json({
+        error: 'This permission is already defined'
+      });
+    }
+    const payload = {
+      roleId: id,
+      permission: newPermission,
+    };
+
+    const createPermission = await Permission.create(payload);
+    return res.status(201).json({
+      permission: {
+        id: createPermission.id,
+        role,
```

Comment:
the data sent back here should be from the created object in the db, hence rather than send back `role`, send back `createdPermission.roleId`

Reply:
Done